webapp/src/ts/services/user-contact.service.ts

```diff
@@ -33,4 +33,17 @@ export class UserContactService {
         throw err;
       });
   }
+
+  async getCurrentLineageLevel(){
```

Comment:
+1

Reply:
Yeah I agree, I unsuccessfully tried to find a better place for it. I would be fine duplicating the function, but as I will need to ask breadcrumbs to Tasks tab as well when working on #5710 , it would make it a tri-plication 🤔